app/pages/preview/template.test.js

```diff
@@ -218,6 +228,39 @@ describe('preview page', () => {
         expect(table.find('[data-test-id="column-heading-5"]').text().trim()).toEqual('Item cost (£)');
       });
     }));
+
+    it('should render the one off cost details', componentTester(setup, (harness) => {
+      harness.request(context, ($) => {
+        const table = $('div[data-test-id="one-off-cost-table"]');
+        const oneOffCost1Row = table.find('[data-test-id="table-row-0"]');
+        const recipientNameDetails = oneOffCost1Row.find('div[data-test-id="recipient-name"]');
+        const itemId = oneOffCost1Row.find('div[data-test-id="item-id"]');
+        const itemName = oneOffCost1Row.find('div[data-test-id="item-name"]');
+        const priceUnit = oneOffCost1Row.find('div[data-test-id="price-unit"]');
+        const quantity = oneOffCost1Row.find('div[data-test-id="quantity"]');
+        const itemCost = oneOffCost1Row.find('div[data-test-id="item-cost"]');
+
+        expect(oneOffCost1Row.length).toEqual(1);
+
+        expect(recipientNameDetails.length).toEqual(1);
```

Comment:
Is this similar to doing the OK checks that were removed? Are the length checks needed? I suppose these are more of a unit test so we aren't affected too much by them.

Reply:
yeah you are right. They are redundant but these are pretty quick to run though. Will leave them in for now.